README.md

```diff
@@ -0,0 +1,19 @@
+# chef
+Task for Epam Java Collection course
+
+In this project you can find salad making functionality. The program contains list of ingredients and two ready salads - Chef's and Russian salads.
```

Comment:
Can you add quick reference to class where ingredients are stored and also where ready salads are stored?

Reply:
Added